devops/ansible-role-girder/tasks/main.yml

```diff
@@ -23,10 +23,11 @@
 - import_tasks: nodejs.yml
   when: girder_web|bool
 
-- name: Upgrade pip in the virtualenv
+- name: Install a recent pip version
   pip:
     name: pip
-    extra_args: "--update"
+    version: ">=21.0.1"
+    state: present
```

Comment:
```suggestion
```

This is the default.

Reply:
Yep.  You asked for it, though.

Reply:
Yeah, if you're using `state: upgrade`, it's necessary, but the minimum version is better for idempotency. Sorry for the confusion from my lack of specificity.

I have a number of CD pipelines using this role, and minimizing unnecessary Ansible changes is an important requirement.